stable/contour/values.yaml

```diff
@@ -0,0 +1,63 @@
+deployment:
+  replicas: 2
+contour:
+  image:
+    repository: gcr.io/heptio-images/contour
+    tag: v0.5.0
+    pullPolicy: Always
```

Comment:
Same as the previous pullPolicy comment

Reply:
Changed image pullpolicy to IfNotPresent